Content.Server/Cargo/Systems/CargoSystem.Orders.cs

```diff
@@ -199,7 +199,7 @@ private void OnAddOrderMessage(EntityUid uid, CargoOrderConsoleComponent compone
 
             // Log order addition
             _adminLogger.Add(LogType.Action, LogImpact.Low,
-                $"{ToPrettyString(player):user} added order [orderNum:{data.OrderNumber}, amount:{data.Amount}, product:{data.ProductId}, requester:{data.Requester}, reason:{data.Reason}]");
+                $"{ToPrettyString(player):user} added order [orderNum:{data.PrintableOrderNumber}, amount:{data.Amount}, product:{data.ProductId}, requester:{data.Requester}, reason:{data.Reason}]");
```

Comment:
This integer doesn't align with vv so use the index.

Reply:
Aha, didn't know what the preference was here. Thanks - will bear that in mind for the future.